lua/lspconfig/server_configurations/drools_lsp.lua

```diff
@@ -0,0 +1,119 @@
+local util = require 'lspconfig.util'
+
+local get = function(config, section, key)
```

Comment:
we have `vim.tbl_get` function to use no need write a new one.

`vim.tbl_get(config, section,key)`

Reply:
Okay, I can change to using that function. Would it be better to add a 2nd commit to this PR, or a just squash the commits and force-push it?

Reply:
add 2nd commit

Reply:
@glepnir Done.